### PR TITLE
Update ns args in SYSTEM messages

### PR DESCRIPTION
**Pull request by mattias-p**
This is a follow-up to #790 with the corresponding updates to the SYSTEM messages.

It is also a step towards solving #713.

### CODE REVIEW

File: lib/Zonemaster/Engine/Translator.pm [hunk 1]
```diff
@@ -97,13 +97,13 @@ Readonly my %TAG_DESCRIPTIONS => (
     },
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Translator.pm:97]: ADDED_FAKE_DELEGATION and FAKE_DELEGATION_TO_SELF, a few lines up, should also be updated.

File: lib/Zonemaster/Engine/Translator.pm [hunk 2]
```diff
@@ -77,11 +77,11 @@ Readonly my %TAG_DESCRIPTIONS => (
...
         __x    # SYSTEM:SKIP_IPV4_DISABLED
-          "IPv4 is disabled, not sending query to {ns}.", @_;
+          "IPv4 is disabled, not sending query to {ns_list}.", @_;
     },
     SKIP_IPV6_DISABLED => sub {
         __x    # SYSTEM:SKIP_IPV6_DISABLED
-          "IPv6 is disabled, not sending query to {ns}.", @_;
+          "IPv6 is disabled, not sending query to {ns_list}.", @_;
     },
     FAKE_DELEGATION => sub {
```
Comment (matsduf) [inline lib/Zonemaster/Engine/Translator.pm:86]: Defined without arguments, but used with. (Just an observation, not for this PR.)

**Review by matsduf — CHANGES_REQUESTED** [review]

File: lib/Zonemaster/Engine/Translator.pm (hunk 1 above)
Comment (mattias-p) [reply lib/Zonemaster/Engine/Translator.pm:97]: In what way should they be updated?

Comment (matsduf) [reply lib/Zonemaster/Engine/Translator.pm:97]: I assumed that "ns" was a name server name, as it has usually been used for, not a name server name / IP address pair. Where ADDED_FAKE_DELEGATION and FAKE_DELEGATION_TO_SELF are used it says `ns => "$self"` and I am not sure what `$self` will return there.

Comment (mattias-p) [reply lib/Zonemaster/Engine/Translator.pm:97]: It evaluates to a name / IP address string.

**Review by matsduf — APPROVED** [review]

**Pull request merged**